float32 left_elbow_angle        # angle at left elbow
float32 right_elbow_angle       # angle at right elbow
float32 left_wrist_angle        # angle from left shoulder to wrist
float32 right_wrist_angle       # angle from right shoulder to wrist
float32 left_shoulder_angle     # angle at left shoulder
float32 right_shoulder_angle    # angle at right shoulder
float32 right_eye_angle         # neck to right eye angle
float32 left_eye_angle          # neck to left eye angle
float32 nose_vec_x              # vector from neck to nose x
float32 nose_vec_y              # vector from neck to nose y
float32 eye_vec_x               # vector between eyes x
float32 eye_vec_y               # vector between eyes y
float32 shoulder_vec_x          # vector between shoulders x
float32 shoulder_vec_y          # vector between shoulders y
float32 right_arm_vec_x         # vector from shoulder to wrist x
float32 right_arm_vec_y         # vector from shoulder to wrist y
float32 left_arm_vec_x          # vector from shoulder to wrist x
float32 left_arm_vec_y          # vector from shoulder to wrist y
float32 neck_x                  # x coord of neck
float32 neck_y                  # y coord of neck